MoreLinq/MoreLinq.csproj

```diff
@@ -133,8 +133,15 @@
     <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
     <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
     <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
+    <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <EmbedUntrackedSources>true</EmbedUntrackedSources>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0-*" PrivateAssets="All"/>
+  </ItemGroup>
+  
```

Comment:
> Can someone explain me why the checks failed ?

As [line 276 of the build log](https://ci.appveyor.com/project/raboof/morelinq/build/1090#L276) says…

```
eclint check -n "**/*.{cs,tt,cmd,sh,md,txt,yml}"
eclint check -w "**/*.{cs,tt,cmd,sh,md,txt,yml,json,sln,csproj,shfbproj}"
MoreLinq/MoreLinq.csproj
    144:01 ❌ unexpected trailing whitespace (EditorConfig trim_trailing_whitespace https://goo.gl/oo8iVb)
```

…there's _trailing whitespace_ on this line that needs to be removed.

Reply:
I know I saw the log, I just wasn't able to see it on visual studio.
After reopening it I saw it

Reply:
We do have an `.editoconfig` that should trim trailing whitespaces in editors that support it:

https://github.com/morelinq/MoreLINQ/blob/e300696bc41696ae8c154df5838ff54dafb3d691/.editorconfig#L5

Otherwise the [Trailing Whitespace Visualizer](https://marketplace.visualstudio.com/items?itemName=MadsKristensen.TrailingWhitespaceVisualizer) extension can help if you are on an older version of Visual Studio 2017 where `.editoconfig` support was absent or weaker.